app/models/ticket_purchase.rb

```diff
@@ -39,10 +39,16 @@ def self.purchase(conference, user, purchases)
   end
 
   def self.purchase_ticket(conference, quantity, ticket, user)
-    purchase = new(ticket_id: ticket.id,
-                   conference_id: conference.id,
-                   user_id: user.id,
-                   quantity: quantity) if quantity > 0
+    if quantity > 0
+      purchase = new(ticket_id: ticket.id,
+                     conference_id: conference.id,
+                     user_id: user.id,
+                     quantity: quantity,
+                     paid: false)
```

Comment:
why not just say `paid: ticket.price_cents.zero?)` ?
Then you don't need that second `if ticket.price_cents.zero?....`